src/base/preferences.cpp

```diff
@@ -1175,6 +1175,16 @@ void Preferences::setShutdownqBTWhenDownloadsComplete(bool shutdown)
     setValue("Preferences/Downloads/AutoShutDownqBTOnCompletion", shutdown);
 }
 
+bool Preferences::dontConfirmAutoExit() const
+{
+    return value("ShutdownConfirmDlg/DontConfirmAutoExit", false).toBool();
```

Comment:
I changed this to prepare it for SettingsStorage conversion without the need to create a mapping in that class.